bot.go

```diff
@@ -156,11 +156,10 @@ func eval(input string, formatMode bool, compactMode bool) string {
 	switch input {
 	case "":
 		fallthrough
-	case "info":
-		fallthrough
 	case "help":
 		res = "💡 Grol bot help: grol bot evaluates grol language fragments, as simple as expressions like `1+1`" +
```

Comment:
I would have used go:embed, to make it easier to maintain, but why not

Reply:
I don't think it's big enough to warrant a template embed (as it needs to stay fairly small anyway given the medium)

![Uploading Screenshot 2024-08-05 at 4.27.26 PM.png…]()

Reply:
if anything I should put a 

helpText := `...stuff in backtick...` instead of the \n

there are no variables in that text so no need for templates yet